sympy/stats/drv.py

```diff
@@ -79,8 +80,15 @@ def compute_cdf(self, **kwargs):
         cdf = Piecewise((cdf, z >= left_bound), (0, True))
         return Lambda(z, cdf)
 
+    def _cdf(self, x):
+        return None
+
     def cdf(self, x, **kwargs):
         """ Cumulative density function """
+        if len(kwargs) == 0:
```

Comment:
`if not kwargs:` would be simpler. An empty dict is falsy, a nonempty one is truthy.